code/__DEFINES/_tick.dm

```diff
@@ -1,7 +1,11 @@
 /// Percentage of tick to leave for master controller to run
 #define MAPTICK_MC_MIN_RESERVE 70
 /// internal_tick_usage is updated every tick by extools
+#ifdef USE_EXTOOLS
 #define MAPTICK_LAST_INTERNAL_TICK_USAGE ((GLOB.internal_tick_usage / world.tick_lag) * 100)
+#else
+#define MAPTICK_LAST_INTERNAL_TICK_USAGE (world.map_cpu)
```

Comment:
Just a note for any maintainer that doesn't know, this currently rounds to the nearest .125. I believe Lummox is fixing this, though I don't know how critical it is.

Reply:
for live rounds it really isnt critical, its 0.125 of a percent of the tick, so 0.00125%